subsys/bluetooth/host/l2cap.c

```diff
@@ -1201,14 +1250,14 @@ void l2cap_chan_seg_sent(struct bt_conn *conn, void *user_data)
 
 	BT_DBG("conn %p chan %p", conn, chan);
 
-	l2cap_chan_le_send_resume(BT_L2CAP_LE_CHAN(chan));
+	l2cap_chan_tx_resume(BT_L2CAP_LE_CHAN(chan));
 }
 
 static int l2cap_chan_le_send(struct bt_l2cap_le_chan *ch, struct net_buf *buf,
 			      u16_t sdu_hdr_len)
 {
 	struct net_buf *seg;
-	int len;
+	int len, ret;
```

Comment:
Why `ret` instead of `err`? Don't we use `err` everywhere else?

Reply:
.. I can see some other use of `ret` in other "send" functions though

Reply:
No specific reason, I could rename it if that matters to you.

Reply:
You know, this sort of comment comes out like a patronizing thing, there is obviously no logical difference between using an err or ret.

Reply:
It wasn't meant like that. I'm just in a fan of consistency. err is the usual name for returned errno defines. makes it easier to read IMO.